src/api/interface.ts

```diff
@@ -130,6 +138,8 @@ declare class Channel {
    * `Error` with a string from the `ChannelError` enumeration.
    */
   public addBroadcastListener(listener: (event: {channel: Channel; context: Context}) => void): Listener;
```

Comment:
I missed this when the original PR came in, but why is the instance-based listener hook up called `addBroadcastListener` where the global-based listener subscription is called `addContextListener`?

Reply:
yes, we discussed this on thursday and are going to make everything 'addContextListener' across the board.